app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cash = {}
+
+    def wrapper(*args, **kwargs) -> int:
+        for item in cash:
+            if f"{func}, {args}" in cash:
+                print("Getting from cache")
+                return cash[f"{func}, {args}"]
+        else:
```

Comment:
There is no need for `else`, the code will not execute after the return anyway. Remove it after removing the loop.